documentation/how-to-guides/deploy_magma_access_gateway.md

```diff
@@ -24,15 +24,20 @@ Magma Access Gateway using [snap](https://snapcraft.io/magma-access-gateway).
 - Ubuntu 20.04 LTS
```

Comment:
I was wondering if we should place the content of this how-to-guide inside of the `README.md`. What do you think?

Reply:
Yes, I feel like it's not obvious how to install magma-access-gateway. For most snaps you just do snap install. Four ours it's a bit more complex. So I think putting it in the README (and maybe in the snapcraft home page) would be helpful. 

We can use the how-to to explain more complex setups. Example, how-to: deploy snap on X hardware.

Reply:
Agreed.
Moved to README.